db/seeds.rb

```diff
@@ -20,4 +20,77 @@
                email: email,
                password:              password,
                password_confirmation: password)
-end
\ No newline at end of file
+end
+
+
+# Make a rhizome
+Rhizome.create!(
+  name: "Test rhizome"
+  )
+
+ParticleDevice.create!(
+  device_id: Faker::Lorem.characters(24),
+  encrypted_access_token: "S3jF/ramSAL8dZ4zzASXeuBunWv6tLn9jzcU4ABEflfGQkrwbM44CpWMtyRhkED/",
+  rhizome_id: 1
+  )
+
+
+# Make an equipment profile
+EquipmentProfile.create!(
+  name: "Test EP")
+
+HeatingElement.create!(
+  pins: {"power_pin": "1", "control_pin": "0"},
+  equipment_profile: EquipmentProfile.find(1) )
+
+TemperatureSensor.create!(
+  pins: {"data_pin": "2", "onewire_index": "10"},
+  equipment_profile: EquipmentProfile.find(1))
+
+Thermostat.create!(
+  equipment_profile: EquipmentProfile.find(1),
+  element: HeatingElement.create!(
+    pins: {"power_pin": "3", "control_pin": "5"},
+    equipment_profile: EquipmentProfile.find(1)
+    ),
+  sensor: TemperatureSensor.create!(
+    pins: {"data_pin": "4", "onewire_index": "9"},
+    equipment_profile: EquipmentProfile.find(1)
+    )
+  )
+  
+
+# Make a schedule and tasks
+Schedule.create!(
+  name: "Test Sched 1",
+  root_task_id: 1
```

Comment:
Does the root_task already exist at this point? It doesn't seem like it...
Schedule has `accepts_nested_attributes_for :tasks` , so you should be able to create the Task that will be the root_task within the Schedule #create, then follow up with a line that sets the root_task_id to the newly created Task (since there's only one you can get the Task using `schedule.tasks.first`). You'd have to do it manually here because we're still not creating that root_task automatically, but eventually we should change it so that assignment happens automagically.